client/my-sites/plugins/plugin-sections/custom.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import NavItem from 'components/section-nav/item';
+import NavTabs from 'components/section-nav/tabs';
+import SectionNav from 'components/section-nav';
+import { sanitizeSectionContent } from 'lib/plugins/utils';
+
+const PluginSectionsCustom = ( { plugin, translate } ) => {
+	if ( ! plugin.description.length ) {
```

Comment:
Good call, addressed in https://github.com/Automattic/wp-calypso/pull/17274/commits/cb4702c6360f10d9f0178c5faf2eb1b573bc7df3.